### PR TITLE
HAWQ-444. The counter for increasing and decreasing memory quota is not correctly subtracted

### PR DESCRIPTION
**Pull request by jiny2**
This code change is to remove the code for the two counters.

### CODE REVIEW

File: src/test/regress/pg_regress.c [hunk 1]
```diff
@@ -91,8 +91,8 @@ static int	port = -1;
 static char *user = NULL;
 static char *srcdir = NULL;
 static _stringlist *extraroles = NULL;
-char *initfile = NULL;
-char *expected_statuses_file = NULL;
+static char *initfile = "./init_file";
```
Comment (sansanichfb) [inline src/test/regress/pg_regress.c]: Just curios, what is the reason of this change?

Comment (jiny2) [inline src/test/regress/pg_regress.c]: Should be a error from git, another irrelevant commit is involved. This is why I close it at once.

Comment (sansanichfb) [inline src/test/regress/pg_regress.c]: Got it, thanks.
